packages/framework/presence/README.md

```diff
@@ -1,6 +1,88 @@
 # @fluid-experimental/presence
 
-A session only DDS for lightweight data sharing.
+A set of session focus utilities for lightweight data sharing and messaging.
+
+A session is a period of time when one or more clients are connected to a Fluid service. Session data and messages may be exchanged among clients, but will disappear once the no clients remain. (More specifically once no clients remain that have acquired the session `IPresence` interface.) Once fully implemented, no client will require container write permissions to use Presence features.
+
+## Concepts
+
+### Attendees
+
+For the lifetime of a session, each client connecting will be established as a unique and stable `ISessionClient`. The representation is stable because it will remain the same `ISessionClient` instance independent of connection drops and reconnections.
+
+Client Ids on maintained by `ISessionClient` may be used associate `ISessionClient` with quorum, audience, and service audience members.
+
+### Workspaces
+
+Within Presence data sharing and messaging is broken into workspaces with custom identifiers (workspace addresses). Clients must use the same address within a session to connect with others. Unique addresses enable logical components within a client runtime to remain isolated or work together (without other threading).
```

Comment:
What does threading mean here?

Reply:
I think piping is the better term. I mean having two pieces of code that are separated by some distance for whatever reason be able to work together. This would mean some schema is established common to multiple pieces of logic - sharing that and the workspace address will enable them to work together on a workspace without piping the workspace object from one to another.
Where is our technical writer?